### ISSUE TITLE
Bootable NixOS ISOs not working (18.03.132847)

### ISSUE THREAD

**Issue by twashing** [NixOS/nixpkgs#44888]
## Issue description

I tried to make a bootable NixOS Installation USB stick. I followed the ["Burning" to USB drive](https://nixos.wiki/wiki/NixOS_Installation_Guide) section, in the NixOS Installation Guide. Both the graphical and minimal iso(s) fail when I try to write the iso to my USB. 

## Steps to reproduce

I tried to make the bootable USB from both my MacOS and a Ubuntu linux systems, following the guide. Both approaches fail in the same way. I tried both of these commands to write the graphical (and minimal) iso(s). And both resulting USB sticks fail. 

**A)** When `dd` has finished, I simply reboot with the inserted USB stick and enter the boot loader. From there, I select the USB stick as a boot launch medium. 

**B)** From there, I'm immediately exited, out of the boot process without any error message. I simply immediately placed back in the boot loader. 

```
sudo dd if=~/Downloads/nixos-graphical-18.03.132847.aec217852f2-x86_64-linux.iso of=/dev/sda
sudo dd bs=1024 status=progress if=~/Downloads/nixos-graphical-18.03.132847.aec217852f2-x86_64-linux.iso of=/dev/sda
```

**Comment by knedlsepp** [comment]
Are you sure you selected the correct device? Try `diskutil list` to get the list of all devices. Identify your USB drive and use the corresponding /dev/diskX instead of /dev/sda.

**Comment by twashing** [comment]
Hey @knedlsepp thanks for getting back to me. I solved this problem. It was a bunch of little issues. But I muddled through sing a few blog posts. 

- ["Installing NixOS"](https://totaltrash.xyz/install-nixos/) helped me create the Bootable NixOS ISO.
- And a different ["Installing NixOS"](https://adelbertc.github.io/posts/2017-08-13-installing-nixos.html), helped me with the actual installation.

Hope that helps
